examples/multiple_windows/lib/app/main_window.dart

```diff
@@ -0,0 +1,259 @@
+  MainWindow({super.key, required BaseWindowController mainController}) {
+    windowManager.add(
+      KeyedWindow(
+        isMainWindow: true,
+        key: UniqueKey(),
+        controller: mainController,
+      ),
+    );
+  }
+
+  final WindowManager windowManager = WindowManager();
+  final WindowSettings settings = WindowSettings();
+
+  @override
+  Widget build(BuildContext context) {
+    return WindowManagerAccessor(
+      windowManager: windowManager,
+      child: WindowSettingsAccessor(
+        windowSettings: settings,
+        child: ViewAnchor(
+          view: ListenableBuilder(
+            listenable: windowManager,
+            builder: (BuildContext context, Widget? _) {
+              final List<Widget> childViews = <Widget>[];
+              for (final KeyedWindow window in windowManager.windows) {
+                if (window.parent == null && !window.isMainWindow) {
+                  childViews.add(
+                    WindowContent(
+                      controller: window.controller,
+                      windowKey: window.key,
+                      onDestroyed: () => windowManager.remove(window.key),
+                      onError: () => windowManager.remove(window.key),
+                    ),
+                  );
+                }
+              }
+
+              return ViewCollection(views: childViews);
+            },
+          ),
+          child: Scaffold(
+            appBar: AppBar(title: const Text('Multi Window Reference App')),
+            body: Row(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                Expanded(
+                  flex: 60,
+                  child: SingleChildScrollView(
+                    scrollDirection: Axis.vertical,
+                    child: _WindowsTable(windowManager: windowManager),
+                  ),
+                ),
+                Expanded(
+                  flex: 40,
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.stretch,
+                    children: [
+                      ListenableBuilder(
+                        listenable: windowManager,
+                        builder: (BuildContext context, Widget? child) {
+                          return _WindowCreatorCard(
+                            selectedWindow: windowManager.selected,
+                            windowManagerModel: windowManager,
+                            windowSettings: settings,
+                          );
+                        },
+                      ),
+                    ],
+                  ),
+                ),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class _WindowsTable extends StatelessWidget {
+  const _WindowsTable({required this.windowManager});
+
+  final WindowManager windowManager;
+
+  @override
+  Widget build(BuildContext context) {
+    return ListenableBuilder(
+      listenable: windowManager,
```

Comment:
Now that we have WindowManagerAccessor, would it be cleaner to use WindowManagerAccessor.of here instead of passing this in? I think you could get rid of the ListenableBuilder then.